README.md

```diff
@@ -126,7 +126,8 @@ to update UUIDs in your Vagrantfile. If both are specified, the id parameter tak
 * `instance_ready_timeout` - The number of seconds to wait for the instance
   to become "ready" in Cloudstack. Defaults to 120 seconds.
 * `domain_id` - Domain id to launch the instance into
-* `network_id` - Network uuid that the instance should use
+* `network_id` - Network uuid(s) that the instance should use
+  - `network_id` is single value (e.g. `"AAAA"`) or multiple values (e.g. `["AAAA", "BBBB"]`)
 * `network_name` - Network name that the instance should use
```

Comment:
@sue445 Nice one! But if we do multiple networks by `network_id`, we should also allow that via `network_name`